src/app/pages/home/home.component.ts

```diff
@@ -4,6 +4,7 @@ import { UserComponent } from '../../features/user/user.component';
 import { BookshelfComponent } from '../../features/bookshelf/bookshelf.component';
 import { ReadingNowComponent } from '../../features/reading-now/reading-now.component';
 import { PurchaseComponent } from '../../features/purchase/purchase.component';
+import { BooksReviewComponent } from '../../features/books-review/books-review.component';
 
 @Component({
```

Comment:
This component is not longer looking like an standalone component, it looks more like Module, it should be a good addition refactor the code and transform this component to a module.